src/innsyn/InnsynRouter.tsx

```diff
@@ -20,7 +19,6 @@ const InnsynRouter: React.FC = () => {
                 <div className={"blokk-center " + (ekstraSpaltebredde ? "blokk-center--wide" : "")}>
                     <Switch>
                         <Route exact path="/innsyn/utbetaling" component={Utbetalinger} />
-                        <Route exact path="/innsyn/demo" component={InnsynDemoMeny} />
```

Comment:
Usikker om FSL bruker denne for å finne swagger. 
Enig i at det er bedre å bare fjerne den så den ikke er i prod.

Reply:
Hvis noen skulle trenge lenken til swagger kan vi heller dele den på slack eller lignende 🙂

Reply:
Enig 👍🏻